### PR TITLE
feat(app/pano):used toast component

### PR DESCRIPTION
**Pull request by muhammed-gumus**
# Description

Added Toast component that will be triggered when the Copy Link text in the DropDown Menu is clicked.

### Checklist

- [x] discord username: `muhammed-gumus`
- [x] Closes <https://github.com/kamp-us/monorepo/issues/568>
- [x] PR must be created for an issue from issues under "In progress" column from [our project board](https://github.com/orgs/kamp-us/projects/2/views/1).
- [x] A descriptive and understandable title: The PR title should clearly describe the nature and purpose of the changes. The PR title should be the first thing displayed when the PR is opened. And it should follow the semantic commit rules, and should include the app/package/service name in the title. For example, a title like "docs(@kampus-apps/pano): Add README.md" can be used.
- [ ] Related file selection: Only relevant files should be touched and no other files should be affected.
- [ ] I ran `npx turbo run` at the root of the repository, and build was successful.
- [ ] I installed the npm packages using `npm install --save-exact <package>` so my package is pinned to a specific npm version. Leave empty if no package was installed. Leave empty if no package was installed with this PR.

### How were these changes tested?

Please describe the tests you did to test the changes you made. Please also specify your test configuration.

### CODE REVIEW

**Review by Ketcap — APPROVED** [review]

File: apps/kampus/app/layout.tsx [hunk 1]
```diff
@@ -1,11 +1,12 @@
 import { type ReactNode } from "react";
 import { Inter } from "next/font/google";
+import { ToastProvider } from "@kampus/ui-next";
 
 import { RelayEnvironmentProvider } from "~/features/relay/RelayEnvironmentProvider";
 
 import "./globals.css";
 
-import { ThemeProvider } from "@kampus/ui-next";
+import { ThemeProvider, Toast } from "@kampus/ui-next";
```
Comment (cansirin) [inline apps/kampus/app/layout.tsx]:
```suggestion
import { ThemeProvider } from "@kampus/ui-next";
```

File: apps/kampus/app/pano/features/post-list/MoreOptions.tsx [hunk 2]
```diff
@@ -54,7 +56,15 @@ export const MoreOptionsDropdown = ({ post, shareUrl }: Props) => {
       </DropdownMenuTrigger>
       <DropdownMenuContent>
         {ownerItems}
-        <DropdownMenuItem>Adresi kopyala</DropdownMenuItem>
+        <DropdownMenuItem
+          onSelect={() => {
+            toast({
+              description: "link kopyalandı",
```
Comment (cansirin) [inline apps/kampus/app/pano/features/post-list/MoreOptions.tsx]:
```suggestion
              description: "Link kopyalandı",
```

File: apps/kampus/app/pano/features/post-list/MoreOptions.tsx [hunk 3]
```diff
@@ -54,7 +56,15 @@ export const MoreOptionsDropdown = ({ post, shareUrl }: Props) => {
...
+            });
+          }}
+        >
+          Adresi kopyala
```
Comment (cansirin) [inline apps/kampus/app/pano/features/post-list/MoreOptions.tsx]:
```suggestion
          Linki kopyala
```

Hangi kelimeyi kullanmamiz gerektigine tam karar veremiyorum, var mi oneriniz? @usirin @Ketcap

Comment (usirin) [reply apps/kampus/app/pano/features/post-list/MoreOptions.tsx]: Linki kopyala daha mantikli, let's go

File: apps/kampus/app/pano/features/post-list/MoreOptions.tsx (hunk 2 above)
Comment (usirin) [reply apps/kampus/app/pano/features/post-list/MoreOptions.tsx]: `Link kopyalandı` iyidir bence.

**Comment by Ketcap** [comment]
If you can make sure to go through checklist I would merge it immediately :)

File: apps/kampus/app/pano/features/post-list/MoreOptions.tsx [hunk 4]
```diff
@@ -25,8 +28,8 @@ type Post = {
 
 export const MoreOptionsDropdown = ({ post, shareUrl }: Props) => {
   // const user = useUserContext();
-  console.log(post, shareUrl);
-
+  //console.log(post, shareUrl);
```
Comment (usirin) [inline apps/kampus/app/pano/features/post-list/MoreOptions.tsx:31]: let's remove this

**Review by usirin — CHANGES_REQUESTED** [review]
Things look pretty good, though we need to run prettier on the changed files. @cansirin can you assist?

**Review by usirin — APPROVED** [review]

File: apps/kampus/app/pano/features/post-list/MoreOptions.tsx (hunk 4 above)
Comment (usirin) [inline apps/kampus/app/pano/features/post-list/MoreOptions.tsx]:
```suggestion
```

**Review by cansirin — APPROVED** [review]
LETS GOOOOOOOO!

**Pull request merged**